com.playeveryware.eos/Runtime/Core/Config/Attributes/ParsesToAuthScopeFieldValidatorAttribute.cs

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2024 PlayEveryWare
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+namespace PlayEveryWare.EpicOnlineServices
+{
+    using Epic.OnlineServices.Auth;
+    using Epic.OnlineServices.Platform;
+    using PlayEveryWare.EpicOnlineServices.Extensions;
+    using System;
+    using System.Collections.Generic;
+    using System.Reflection;
+    using System.Text;
+    using System.Text.RegularExpressions;
+
+    [AttributeUsage(AttributeTargets.Field)]
+
+    public class ParsesToAuthScopeFieldValidatorAttribute : ParsesToEnumFieldValidatorAttribute
+    {
+        protected override bool CanParse(List<string> flags)
+        {
+            if (!AuthScopeFlagsExtensions.TryParse(flags, out AuthScopeFlags results))
+            {
+                return false;
+            }
+
+            return true;
+        }
```

Comment:
This should no longer be needed - since the `List<string>` type of Enums disappeared for `EOSConfig`.

Reply:
This can be removed, because it is specific on AuthScopeFlags

Reply:
Actually it can't because it needs to implement CanParse from its base

Reply:
I think that `ParsesToEnumFieldValidatorAttribute` can be removed altogether - there should not be a need for it since there are no longer any `enum` values that are represented by `List<string>`.